src/sentry/eventstream/kafka/backend.py

```diff
@@ -127,6 +137,10 @@ def insert(self, group, event, is_new, is_sample, is_regression,
             organization=project.organization,
         )
 
+        unexpected_tags = set([k for (k, v) in event.get_tags() if k in self.UNEXPECTED_TAG_KEYS])
```

Comment:
`event.get_tags` has more overhead than needed here but I didn't want to copy the logic there

Reply:
do we actually want to check `event.get_raw_data()['tags']` ? I guess `get_tags()` doesn't really do anything else substantial to the internal event.data, but if I put on my pedant hat, this isn't technically the same thing as we're sending. 

Definitely very minor nit though, I'm pretty sure this will catch everything that we actually need to catch

Reply:
> do we actually want to check `event.get_raw_data()['tags']` ? I guess `get_tags()` doesn't really do anything else substantial to the internal event.data, but if I put on my pedant hat, this isn't technically the same thing as we're sending.

Made this more explicit/marginally more future-proof with c276fdabd09bcd4cdf8eb88ffcaf7a0cd7bdd963